instance-scheduler/main.go

```diff
@@ -324,18 +473,36 @@ func handler(request InstanceSchedulingRequest) (events.APIGatewayProxyResponse,
 	accounts := getNonProductionAccounts(environments, skipAccounts)
 	memberAccountNames := []string{}
 	nonMemberAccountNames := []string{}
-	totalCount := &InstanceCount{actedUpon: 0, skipped: 0, skippedAutoScaled: 0}
+	totalCount := &InstanceSchedulingResponse{
+		Action:                request.Action,
+		MemberAccountNames:    []string{},
+		NonMemberAccountNames: []string{},
+		ActedUpon:             0,
+		Skipped:               0,
+		SkippedAutoScaled:     0,
+		RDSActedUpon:          0,
+		RDSSkipped:            0,
+		RDSSkippedAutoScaled:  0,
+	}
 	for accName, accId := range accounts {
 		ec2Client := getEc2ClientForMemberAccount(cfg, accName, accId)
-		if ec2Client == nil {
+		rdsClient := getRDSClientForMemberAccount(cfg, accName, accId)
+
+		if ec2Client == nil && rdsClient == nil {
```

Comment:
this should be "or", as if one of the clients cannot be fetched, it means that it is non-member account

Reply:
corrected to or